tests/opentracer/test_tracer.py

```diff
@@ -304,27 +304,20 @@ def trace_one():
             with ot_tracer.start_active_span(str(id)):
                 event.set()
                 id += 1
-                event.wait()
                 with ot_tracer.start_active_span(str(id)):
-                    event.set()
                     id += 1
-                    event.wait()
                     with ot_tracer.start_active_span(str(id)):
-                        event.set()
```

Comment:
And we should keep this one.

But can keep the `event.wait()` where it is for `trace_two`.

That means, `trace_two` will sit and wait until all the spans from `trace_one` have finished before it starts.


Assuming that is what we are trying to test?